stdlib/REPL/src/docview.jl

```diff
@@ -208,7 +208,7 @@ function lookup_doc(ex)
     end
     if isa(ex, Symbol) && Base.isoperator(ex)
         str = string(ex)
-        if endswith(str, "=")
+        if endswith(str, "=") && ex ∉ (:(==), :(===), :(!=), :(!==), :(<=), :(>=), :(.==), :(.===), :(.!=), :(.!==), :(.<=), :(.>=))
```

Comment:
You could ask if `Base.operator_precedence(ex) == Base.prec_assignment` perhaps

Reply:
Thanks, that's a much more elegant alternative! I just implemented that in the new commit